e2e/src/app.e2e-spec.ts

```diff
@@ -152,27 +154,35 @@ describe('Test App', () => {
 
             await addPropButton.click();
 
-            expect(await submitButton.isDisabled()).toBe(true);
+            const submitButton2 = await page.getAdvancedSearchSubmitButton(loader);
+
+            expect(await submitButton2.isDisabled()).toBe(true);
 
             const selectProps = await page.getAdvancedSearchPropertySelection(loader);
 
             await selectProps.open();
 
             await selectProps.clickOptions({text: 'Integer'});
 
-            expect(await submitButton.isDisabled()).toBe(true);
+            const submitButton3 = await page.getAdvancedSearchSubmitButton(loader);
+
+            expect(await submitButton3.isDisabled()).toBe(true);
 
             const selectCompOps = await page.getAdvancedSearchComparisonOperatorSelection(loader);
 
             await selectCompOps.clickOptions({ text: 'is equal to'});
 
-            expect(await submitButton.isDisabled()).toBe(true);
+            const submitButton4 = await page.getAdvancedSearchSubmitButton(loader);
+
+            expect(await submitButton4.isDisabled()).toBe(true);
 
             const input = await loader.getHarness(MatInputHarness);
 
             await input.setValue('1');
 
-            expect(await submitButton.isDisabled()).toBe(false);
+            const submitButton5 = await page.getAdvancedSearchSubmitButton(loader);
```

Comment:
@mdelez I had to add this workaround because the existing instance always returned the old value for the state of the disabled prop.

I believe that we had a similar problem once but there the element was recreated in the DOM which is not the case here. Fishing in the dark here ...

Reply:
yeah from what I can remember, it won't update automatically if the value changes but I think you can reuse the same variable, no?

Reply:
> but I think you can reuse the same variable, no?

Yes, good point. I think. I could even avoid using a variable in the first place, see https://github.com/dasch-swiss/dsp-ui-lib/pull/290/commits/7e21377cabaf8d5e24b1a5a3beef0a2b512d43e1

This failed locally, let's see how it behaves here.